go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -74,7 +77,20 @@ func (agent *ActionAgent) Backup(ctx context.Context, concurrency int, logger lo
 		if err := agent.refreshTablet(ctx, "before backup"); err != nil {
 			return err
 		}
+	} else {
+		if agent._isOnlineBackupRunning {
```

Comment:
This should be done while holding the lock. Same thing while exporting stats. And same comments in `Backup`

Reply:
+1 but be careful to release the lock before returning. I recommend a pair of helpers like `agent.beginOnlineBackup()` and `agent.endOnlineBackup()` to handle the locking and stats update. Then you can use `defer agent.mutex.Unlock()` inside each helper, and I think you can also `defer agent.endOnlineBackup()` here so it's guaranteed for any return path.

Reply:
I have called them `beginBackup` and `endBackup`. we set `_isBackupRunning` to `true` or `false` and then export the right value in the stats so that regardless of online/offline, we can see the stats in /debug/vars or prometheus metrics.